_posts/en/newsletters/2020-07-01-newsletter.md

```diff
@@ -0,0 +1,355 @@
+  ([transcript][watchtowers ts], [video][watchtowers vid],
+  [slides][watchtowers slides])
+
+- **CoinSwap:** Aviv Milner presented on CoinSwap at the Wasabi Research
+  Club. He explained the property of covertness and how Chris Belcher’s
+  CoinSwap proposal (see [Newsletter #100][news100 coinswap]) provides
+  covertness in a manner that other privacy schemes such as
+  [coinjoin][topic coinjoin] fail to do. Milner also went through the
+  motivation for routing CoinSwaps, namely to address the concern of
+  unwittingly entering into a CoinSwap with an adversary such a chain
+  surveillance company.  ([transcript][coinswap ts], [video][coinswap
+  vid])
+
+- **Taproot and schnorr multisignatures:** A wide ranging discussion at
+  London Bitcoin Devs on the [BIP340][] specification of [schnorr
+  signatures][topic schnorr signatures] with some of the co-authors such as
+  Tim Ruffing, Pieter Wuille, and Jonas Nick.  This covered some of the
+  earlier ideas for implementing schnorr in Bitcoin and what the
+  co-authors thought the community should be concerned about when
+  considering a possible future soft fork deployment. Wuille expressed
+  his view that it is unlikely at this stage that there are bugs in the
+  proposed consensus implementation and that he is more concerned with
+  how schnorr is adopted and what is built using it.
+  ([transcript][socratic ts], [video][socratic vid])
+
+    The following day, Tim Ruffing presented on multisignature and
+    threshold signature schemes using schnorr signatures. He highlighted
+    how the rogue key attack and an attack using Wagner’s algorithm (see
+    [Newsletter #74][news74 wagner]) need to be addressed when designing
+    secure multisignature schemes.  Ruffing has been working with
+    collaborators on a speculative [MuSig][topic musig] signature scheme
+    that only requires two rounds of interaction rather than three
+    (without relying on zero knowledge proofs). Threshold signature
+    schemes generally present further challenges such as requiring an
+    honest majority, a broadcast channel, and careful consideration of
+    parallel signing sessions.  ([transcript][ruffing ts],
+    [video][ruffing vid], [slides][ruffing slides])
+
+
+- **Sydney meetup discussion:** A number of Bitcoin developers and LN
+  developers joined this Sydney meetup to discuss various topics. Ruben
+  Somsen gave a short presentation on Succinct Atomic Swaps (SAS), see
+  [Newsletter #98][news98 sas].  Somsen explained how SAS compared to
+  Chris Belcher’s CoinSwap proposal, whether atomic swaps via privacy
+  focused altcoins is a workable approach to obtain additional privacy,
+  and how SAS only requires one of the two block chains to have timelocks
+  and scripting opcodes.
+
+    Other non-related topics included reliance on DNS seeds in Bitcoin
+    Core, whether they present a viable attack vector against new full
+    nodes seeking to locate network peers for the very first time, and
```

Comment:
nit: I think "and **how** work by Matt Corallo and Antoine Riard on downloading block headers over HTTP and allowing other alternative transports ~~that~~ could potentially mitigate this risk" reads better.

Please also note that this work was brought up briefly as possible mitigations but not discussed in any depth.